### PR TITLE
EZP-30376 [Travis] Implementing reporting tool to behat automated tests

### PR DESCRIPTION
**Pull request by m-tyrala**
[EZP-30376](https://jira.ez.no/browse/EZP-30376#)

- adding RSA key for sending Allure report to the reports server
- encrypting/decrypting key
- adding Allure report extension
- adding `after_script` section with test report processing

TODO before merge:
- [ ] squash commits
- [ ] remove temporary changes for testing related PRs

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -71,12 +81,14 @@
...
         "ezsystems/behat-screenshot-image-driver-cloudinary": "^1.1.0@dev",
-        "ezsystems/behatbundle": "^7.0.0@dev",
+        "ezsystems/behatbundle": "dev-EZP-30376 as 7.0.x-dev",
         "overblog/graphiql-bundle": "^0.1.2",
         "phpunit/phpunit": "^6.5.13",
         "sensio/generator-bundle": "^3.1.7",
         "symfony/phpunit-bridge": "^3.4.18",
-        "liuggio/fastest": "^1.6"
+        "liuggio/fastest": "^1.6",
+        "ezsystems/allure-behat": "^2.0",
```
Comment (mnocon) [inline composer.json]: Packages should be added in alphabetical order, as this limits conflicts on upstream merges.

File: .travis.yml [hunk 2]
```diff
@@ -89,6 +91,7 @@ install:
 before_script:
   # Execute Symfony command if injected into test matrix
   - if [ "${SYMFONY_CMD}" != "" ] ; then docker-compose exec --user www-data app sh -c "bin/console ${SYMFONY_CMD}" ; fi
+  - cd "$HOME/build/ezplatform"; composer why jms/serializer;
```
Comment (mnocon) [inline .travis.yml]: Remember to remove 😉

**Comment by m-tyrala** [comment]
@lserwatka this one can be merged

**Comment by lserwatka** [comment]
You can merge it up.

**Comment by m-tyrala** [comment]
Will be done right after clarifying things with merging `3f7a69041c8596178fd834f74a5b741a4f0c9c52`

**Pull request merged**